java/ql/lib/semmle/code/java/security/performance/RegExpTreeView.qll

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module should provide a class hierarchy corresponding to a parse tree of regular expressions.
```

Comment:
"should"? I guess hopefully it does. This is the interface to the shared redos libs, right?, so it should probably say so.

Reply:
"should" in the sense of "it needs to" rather than "I think it does"